pkgs/development/python-modules/backports-entry-points-selectable/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, importlib-metadata
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "backports.entry-points-selectable";
+  version = "1.0.3";
+
+  src = fetchPypi {
+    inherit version;
+    # pypi project name and tarball name differ
+    pname = builtins.replaceStrings [ "-" ] [ "_" ] pname;
+    sha256 = "f30bcd19c5e2728ac93821d2b6ae0a325597e0ca12324fd91a39fa80e1cd0dd8";
+  };
+
+  format = "pyproject";
+
+  propagatedBuildInputs = lib.optionals (pythonOlder "3.8") [
+    importlib-metadata
+  ];
+
+  checkInputs = [
+    pytestCheckHook
+  ];
+
+  meta = with lib; {
```

Comment:
Also this has an enabled test suite.

Reply:
> pkgutil is a standard library?

not in nixos.

> Also this has an enabled test suite.

Adding a pythonImportsCheck doesn't hurt and could show a missing setuptools.

Reply:
```
nix-shell -p python3 --run "python -c 'import pkgutil'"
```

succeeds, so seems to be a standard library.

Seems like the only thing we may need to propagate is pytest.

Reply:
> Seems like the only thing we may need to propagate is pytest.

pytest shouldn't be propagated because it makes it harder to change the used pytest version in a downstream package.